client/analytics/report/orders/table.js

```diff
@@ -192,7 +192,7 @@ export default class OrdersReportTable extends Component {
 						formattedCoupons.length ? [ formattedCoupons[ 0 ] ] : [],
 						formattedCoupons
 					),
-					value: formattedCoupons.map( item => item.code ).join( ' ' ),
+					value: formattedCoupons.map( coupon => coupon.label ).join( ' ' ),
```

Comment:
Can we comma separate these?

```suggestion
					value: formattedCoupons.map( coupon => coupon.label ).join( ', ' ),
```

Reply:
Yea, for sure, I thought about doing that but the **Product** column also had them separated by spaces.